source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -5,7 +5,7 @@
 How it works
 ============
 
-To be able to detect vulnerabilities, now agents are able to natively collect a list of installed applications, sending it periodically to the manager (where it is stored in local sqlite databases, one per agent). Also, the manager builds a global vulnerability database, from publicly available CVE repositories, using it later to cross-correlate this information with the agent's applications inventory data.
+To be able to detect vulnerabilities, now agents are able to natively collect a list of installed applications (:ref:`System inventory <syscollector>`), sending it periodically to the manager (where it is stored in local sqlite databases, one per agent). Also, the manager builds a global vulnerability database, from publicly available CVE repositories, using it later to cross-correlate this information with the agent's applications inventory data.
```

Comment:
Maybe it's time to eliminate the **now** expression, as we aren´t explaining a new functionality when talking about syscollector.

Reply:
I agree